remote-access/ssh/passwordless.md

```diff
@@ -20,7 +20,7 @@ To generate new SSH keys enter the following command:
 ssh-keygen
 ```
 
-Upon entering this command, you'll be asked where to save the key. We suggest you save it in the default location (`/home/pi/.ssh/id_rsa`) by just hitting `Enter`.
+Upon entering this command, you'll be asked where to save the key. We suggest you save it in the default location (`~/.ssh/id_rsa`) by just hitting `Enter`.
```

Comment:
I think that sentence could do with rewording to make it sound a bit more 'professional', for example:

'Upon entering this command, you will be asked where to save the key. We suggest saving it in the default location ('~/.ssh/id_rsa`) by pressing `Enter`.

Reply:
Sorry - please ignore the above comment - the page has multiple similar issues. I will file a separate PR once this integrates.

Reply:
That's the joy of documentation written by multiple different people, each with their own styles :wink: